pkg/cloud/cloud.go

```diff
@@ -247,3 +320,24 @@ func parseEfsTags(tagMap map[string]string) []*efs.Tag {
 	}
 	return efsTags
 }
+
+func getAvailableMountTargets(mountTargets []*efs.MountTargetDescription) []*efs.MountTargetDescription {
+	availableMountTargets := []*efs.MountTargetDescription{}
+	for _, mt := range mountTargets {
+		if *mt.LifeCycleState == "available" {
+			availableMountTargets = append(availableMountTargets, mt)
+		}
+	}
+
+	return availableMountTargets
+}
+
+func getMountTargetForAz(mountTargets []*efs.MountTargetDescription, azName string) *efs.MountTargetDescription {
+	for _, mt := range mountTargets {
+		if *mt.AvailabilityZoneName == azName {
+			return mt
+		}
+	}
+	klog.Infof("There is no mount target match %v. Will pick a random mount target.", azName)
```

Comment:
this message could be moved to where the random mount target is picked (the caller)

Reply:
`azName` is optional and this func is only called if azName is provided. I think we should keep this message here to avoid if-else in the caller to print this message

Reply:
i mean here: https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/470/files#diff-d7ef58b21290a74508c38ca5fdcd104ca5d098820cfac0681ba5415c9820c91dR271